### PR TITLE
User session type credential

### PR DESCRIPTION
**Pull request by AlviseSembenico**

### CODE REVIEW

**Comment by alextreme** [comment]
@AlviseSembenico a lot better, thanks!

Do note that I think we don't want to overwrite the authentication header if it is supplied by the client. The 'session type' authentication should only be used if the client ignores supplying the header in my eyes.

Also feel free to implement the third option for AuthenticationChoices

**Comment by AlviseSembenico** [comment]
@alextreme Updated.

**Comment by AlviseSembenico** [comment]
@alextreme added the header option

**Comment by alextreme** [comment]
@AlviseSembenico check the conflicts & tests, I'll recheck this on monday

**Comment by AlviseSembenico** [comment]
@alextreme Fixed, everything fine.

File: src/vng/testsession/api_views.py [hunk 1]
```diff
@@ -240,6 +257,13 @@ def get_http_header(self, request, endpoint):
             if header.lower() not in whitelist:
                 request_headers[header] = value
 
+        if session.session_type.authentication == choices.AuthenticationChoices.jwt:
+            jwt_auth = get_jwt(session.session_type).credentials()
+            for k, i in jwt_auth.items():
+                request_headers[k] = i
+        elif session.session_type.authentication == choices.AuthenticationChoices.header:
```
Comment (alextreme) [inline src/vng/testsession/api_views.py:264]: @AlviseSembenico I think you still need to check if you're not overwriting the authorization header here.

File: src/vng/testsession/models.py [hunk 2]
```diff
@@ -21,11 +21,15 @@
 
 class SessionType(models.Model):
 
-    name = models.CharField('Naam', max_length=200, unique=True)
-    standard = models.CharField('Standaard', max_length=200, null=True)
-    role = models.CharField('Rol', max_length=200, null=True)
-    application = models.CharField('Applicatie', max_length=200, null=True)
-    version = models.CharField('Versie', max_length=200, null=True)
+    name = models.CharField(max_length=200, unique=True)
```
Comment (alextreme) [inline src/vng/testsession/models.py:24]: @AlviseSembenico Please re-instate my Dutch verbose_names here

File: src/vng/testsession/api_views.py (hunk 1 above)
Comment (alextreme) [reply src/vng/testsession/api_views.py:264]: (and add a test to verify this)

**Pull request merged**